src/main/java/edu/kis/powp/jobs2d/drivers/adapter/AdapterDraw.java

```diff
@@ -26,8 +26,10 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
-
-		drawLine(line);
+        setPosition(x,y);
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.